packages/telepathy_logger.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Telepathy_logger < Package
+  description 'A library for GLib-based Telepathy components.'
+  homepage 'https://telepathy.freedesktop.org/'
+  version '0.8.2'
+  compatibility 'all'
+  source_url 'https://telepathy.freedesktop.org/releases/telepathy-logger/telepathy-logger-0.8.2.tar.bz2'
+  source_sha256 '8fcad534d653b1b365132c5b158adae947810ffbae9843f72dd1797966415dae'
+
+  depends_on 'dbus_glib'
+  depends_on 'libx11'
+  depends_on 'gobject_introspection'
+  depends_on 'telepathy_glib'
+  depends_on 'sqlite'
+  depends_on 'python27' => :build
```

Comment:
`sqlite` and `python27` are core packages?

Reply:
SQLite depends on most, i'll remove it though. Python2.7 is depreciated and will likely eventually need to be removed from the core package list. It wouldn't hurt to start listing it in the dependecy lists of packages.

Reply:
Python 2 is not going away anytime soon so it should be fine to remove.